packages/lesswrong/components/posts/PostsPage/PostsPagePostHeader.tsx

```diff
@@ -334,31 +284,12 @@ const PostsPagePostHeader = ({post, answers = [], dialogueResponses = [], showEm
       </AnalyticsContext>
     </span>
 
-  let secondaryInfoNode;
-  if (isBookUI) {
```

Comment:
I was wondering if this component is now FriendlyUI-only, in which case it may be worth renaming the file (or at least writing a comment at the top about this). However, it looks like this is still used in `PostsCompareRevisions`, which I think is used by all sites? I couldn't find an example post with major revisions to check this case, but you might want to check it for LW.